doc/fuzzing.md

```diff
@@ -131,10 +142,14 @@ You may also need to take care of giving the correct path for `clang` and
 `clang++`, like `CC=/path/to/clang CXX=/path/to/clang++` if the non-systems
 `clang` does not come first in your path.
 
-Full configure that was tested on macOS with `brew` installed `llvm`:
+Full configuration step that was tested on macOS with `brew` installed `llvm`:
 
 ```sh
-./configure --enable-fuzz --with-sanitizers=fuzzer,address,undefined CC=$(brew --prefix llvm)/bin/clang CXX=$(brew --prefix llvm)/bin/clang++
+$ cmake -B build_fuzz \
+   -DCMAKE_C_COMPILER="$(brew --prefix llvm)/bin/clang" \
+   -DCMAKE_CXX_COMPILER="$(brew --prefix llvm)/bin/clang++" \
+   -DBUILD_FOR_FUZZING=ON \
+   -DSANITIZERS=undefined,address,fuzzer \
```

Comment:
do we need the escape at the end?

Reply:
cc @dergoegge

Reply:
We probably don't.

Reply:
Addressed in https://github.com/hebasto/bitcoin/pull/316.

Reply:
Resolved in https://github.com/bitcoin/bitcoin/pull/30454#issuecomment-2281111100.